### ISSUE TITLE
Investigate test failure for demultiplex command

### ISSUE THREAD

**Issue by kvg** [broadinstitute/longbow#119]
The integration test for demultiplex is failing.  It seems to not like data from a pipe for some reason.

**Comment by jamestwebber** [comment]
This was just a little bug--the test is using both an input file and a piped-in input. Going from

```diff
- [ sys.executable, "-m", "longbow", "demultiplex", "-d", "YN", "-o", "demux", annotated_bam_file_from_pipeline ],
+ [ sys.executable, "-m", "longbow", "demultiplex", "-d", "YN", "-o", "demux"],
```

And it passes. Will fix this while working on #118

**Comment by jamestwebber** [comment]
closed by #165
